### PR TITLE
Break Unused Loop

### PR DESCRIPTION
**Pull request by KamuiKody**
**Describe Pull request**
breaks the unused loop and calls it in a function

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/] (Be honest)
- Does your code fit the style guidelines? [yes/]
- Does your PR fit the contribution guidelines? [yes/]

### CODE REVIEW

**Review by mrdiamonddirt — APPROVED** [review]
checked, Works

**Comment by GhzGarage** [comment]
Linting error

**Comment by tom-osborne** [comment]
Rather than creating a new variable called `looped` I think you can make use of the .active attribute to achieve the same thing. It looks like you're just repeating that check with a new variable?

File: client/main.lua [hunk 1]
```diff
@@ -64,19 +64,27 @@ Skillbar.Repeat = function(data)
...
+                        SendNUIMessage({
+                            action = "check",
+                            data = Skillbar.Data.Data,
+                         })
+                    end
+                else
+                    looped = false
+                    break
+                end
+            Wait(1)
```
Comment (D4isDAVID) [inline client/main.lua:83]:
```suggestion
                Wait(1)
```
Fix indentation

File: client/main.lua [hunk 2]
```diff
@@ -64,19 +64,27 @@ Skillbar.Repeat = function(data)
...
+                    if IsControlJustPressed(0, 38) then
+                        SendNUIMessage({
+                            action = "check",
+                            data = Skillbar.Data.Data,
+                         })
+                    end
+                else
+                    looped = false
+                    break
+                end
```
Comment (D4isDAVID) [inline client/main.lua:82]:
```suggestion
            while Skillbar.Data.Active do
                if IsControlJustPressed(0, 38) then
                    SendNUIMessage({
                        action = "check",
                        data = Skillbar.Data.Data,
                    })
                end
```
Remove extra condition check and unused variable

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

**Review by D4isDAVID — APPROVED** [review]
LGTM, tested and works

**Comment by izolox** [comment]
Works fine for me

**Review by tom-osborne — APPROVED** [review]

**Pull request merged**